package.json

```diff
@@ -92,7 +92,7 @@
     "eslint-config-skyscanner-with-prettier": "^0.8.0",
     "eslint-plugin-flowtype": "^3.2.0",
     "eslint_d": "^7.2.0",
-    "flow-bin": "^0.78.0",
+    "flow-bin": "^0.86.0",
```

Comment:
Upgraded Flow to 86 to match the version used in RN 0.58.5: https://github.com/facebook/react-native/blob/v0.58.5/.flowconfig#L101